stubs/PyYAML/yaml/cyaml.pyi

```diff
@@ -33,6 +33,12 @@ class CLoader(CParser, SafeConstructor, Resolver):
 class CSafeLoader(CParser, SafeConstructor, Resolver):
     def __init__(self, stream: str | bytes | _Readable) -> None: ...
 
+class CFullLoader(CParser, FullConstructor, Resolver):
+    def __init__(self, stream: str | bytes | _Readable) -> None: ...
+
+class CUnsafeLoader(CParser, UnsafeConstructor, Resolver):
+    def __init__(self, stream: str | bytes | _Readable) -> None: ...
+
 class CDangerLoader(CParser, Constructor, Resolver): ...  # undocumented
```

Comment:
This was removed between PyYAML 4.1 and 4.2. Should we still keep it in the stubs?

Reply:
I'd remove it. It's really old and undocumented. Seems like if I had some code that relied on it I'd probably appreciate my type checker complaining.